### PR TITLE
TF breakup + Magic

### PR DESCRIPTION
**Pull request by dgerblick**
- Added smoothing to the camera's position (with magic).
- Still outputs transforms in 3d though, next goal is to transform into 2d.
- Seems to be slightly smoother between cameras
- Added Eigen as a dependency for Linear Algebra magic (PR is mainly to trigger the autobuild for the integration test)

### CODE REVIEW

**Comment by jrbaxter0** [comment]
@dgerblick @ajernandes Can you add the changes from today, then I'll approve?

**Comment by jrbaxter0** [comment]
Oh, I see its already there

File: docker/Dockerfile [hunk 1]
```diff
@@ -51,6 +51,16 @@ RUN pip3 install pybullet stable-baselines3 && \
     ros-${ROS_DISTRO}-pointgrey-camera-driver && \
     rm -rf /var/lib/apt/lists/* /root/.cache/pip/
 
+# Install eigen
+RUN mkdir eigen && \
```
Comment (jrbaxter0) [inline docker/Dockerfile:55]: @dgerblick can't we just get this through APT? seems like overkill

File: docker/Dockerfile.local [hunk 2]
```diff
@@ -1,4 +1,5 @@
-FROM purduearc/rocket-league
+ARG TAG=local
+FROM purduearc/rocket-league:${TAG}
```
Comment (jrbaxter0) [inline docker/Dockerfile.local:2]: Is this meant to be here, or just for local testing?

**Review by jrbaxter0 — CHANGES_REQUESTED** [review]
Looks good, but address two questions pls

File: docker/Dockerfile (hunk 1 above)
Comment (dgerblick) [reply docker/Dockerfile:55]: Didn't realize apt had eigen, oops

File: docker/Dockerfile.local (hunk 2 above)
Comment (dgerblick) [reply docker/Dockerfile.local:2]: Typo, line 1 is supposed to be latest. So you can run ./docker-run --build-arg TAG=69 and the docker image for this PR

**Review by jrbaxter0 — APPROVED** [review]

**Pull request merged**